musicbrainz/MusicBrainzCLI/getArtists.js

```diff
@@ -0,0 +1,104 @@
+var request = require("request");
+var fs = require('fs');
+var intervalId3 = null;
+
+var newObj3 = {};
+var artistsAPI = [];
+
+module.exports = {
+  getArtists: function(finalArray) {
+
+    var elemNum = 0;
+
+    var newObj = {
+        table: []
+    };
+
+ intervalId3 = setInterval(function () {
+
+        if (elemNum == finalArray.length) {
+
+            clearInterval(intervalId3);
+
+            obj = newObj;
+
+            for (i = 0; i < obj.table.length; i++) {
+              if (obj.table[i].hasOwnProperty("aboutartist")) {
+                var newObj3 = {
+                    nationality: null,
+                    dateOfBirth: null,
+                    dateOfDeath: null,
+                    placeOfBirth: null,
+                    placeOfDeath: null,
+                    instrument: null,
+                    psuedonym: null,
+                    work: null,
+                    release: null,
+                    tag: null
+                };
+                var aboutArtist = obj.table[i].aboutartist;
+                newObj3.name = aboutArtist.name;
+                newObj3.source_link = "https://musicbrainz.org/artist/" + obj.table[i].artistId;
+
+                if (aboutArtist.hasOwnProperty("life-span")) {
+                    if (aboutArtist["life-span"].hasOwnProperty("begin") && aboutArtist["life-span"].begin != null) {
+                        newObj3.dateOfBirth = aboutArtist["life-span"].begin;
+                    }
+                    if (aboutArtist["life-span"].hasOwnProperty("end") && aboutArtist["life-span"].end != null) {
+                        newObj3.dateOfDeath = aboutArtist["life-span"].end;
+                    }
+                }
+
+                if (aboutArtist.hasOwnProperty("area") && aboutArtist.area != null) {
+                    newObj3.placeOfBirth = aboutArtist.area.name;
+                }
+
+                if (aboutArtist.hasOwnProperty("end_area") && aboutArtist["end_area"] != null) {
+                    newObj3.placeOfDeath = aboutArtist["end_area"].name;
+                }
+
+                artistsAPI.push(newObj3);
+              }
+            }
+
+            json2 = JSON.stringify(artistsAPI); //convert it back to json
+            fs.writeFile('BrainzArtistsSequelize.json', json2, 'utf8', function writeFileCallback(err, data) {
+                process.send("done from Artists");
+                process.exit();
```

Comment:
process.send & process.exit need to be removed as the server.js script in musicbrainz/musicbrainzcli/ does not execute them as a different process (this just kills the whole server.js script)